Modules/feed/engine/MysqlMemory.php

```diff
@@ -2,12 +2,12 @@
 
 class MysqlMemory extends MysqlTimeSeries
 {
-    public function create($feedid, $options)
+    public function create($feedid)
```

Comment:
will removal of 2nd param here cause an error?

Reply:
`Generic.CodeAnalysis.UnusedFunctionParameter.FoundInExtendedClassAfterLastUsed` is the rule being flagged here...